apps/ui/src/components/FormAuctionBid.vue

```diff
@@ -224,86 +205,229 @@ function handlePlaceOrder() {
   });
 }
 
+function handlePriceUpdate(value: string, fromSlider = false) {
+  bidPrice.value = value;
+
+  const price = parseFloat(value);
+  if (!price || !props.totalSupply) {
+    bidFdv.value = '';
+    return;
+  }
+
+  const totalSupplyFormatted = parseFloat(
+    formatUnits(props.totalSupply, props.auction.decimalsAuctioningToken)
+  );
+
+  const fdv = price * totalSupplyFormatted;
+
+  bidFdv.value = fdv.toFixed(AMOUNT_DECIMALS);
+
+  if (!fromSlider) {
+    sliderValue.value = convertPriceToPercentage(price);
+  }
+}
+
+function handleFdvUpdate(value: string) {
+  bidFdv.value = value;
+
+  const fdv = parseFloat(value);
+  if (!fdv || !props.totalSupply) {
+    bidPrice.value = '';
+    return;
+  }
+
+  const totalSupplyFormatted = parseFloat(
+    formatUnits(props.totalSupply, props.auction.decimalsAuctioningToken)
+  );
+
+  const price = fdv / totalSupplyFormatted;
+  sliderValue.value = convertPriceToPercentage(price);
+
+  bidPrice.value = price.toFixed(AMOUNT_DECIMALS);
+}
+
+function handleSliderChange(event: Event) {
+  if (event.target instanceof HTMLInputElement === false) return;
+
+  sliderValue.value = Number(event.target.value);
+
+  const price = convertPercentageToPrice(sliderValue.value);
+  handlePriceUpdate(price.toFixed(AMOUNT_DECIMALS), true);
+}
+
 onMounted(() => {
   const clearingPrice = parseFloat(props.auction.currentClearingPrice);
   if (clearingPrice <= 0) return;
 
-  bidPrice.value = (clearingPrice * DEFAULT_PRICE_PREMIUM).toFixed(
-    PRICE_DECIMALS
+  handlePriceUpdate(
+    (clearingPrice * DEFAULT_PRICE_PREMIUM).toFixed(AMOUNT_DECIMALS)
   );
 });
-
-function togglePriceMode() {
-  const currentPrice = parseFloat(bidPrice.value) || 0;
-  isPriceInverted.value = !isPriceInverted.value;
-  if (currentPrice) {
-    const decimals = isPriceInverted.value
-      ? INVERTED_PRICE_DECIMALS
-      : PRICE_DECIMALS;
-    bidPrice.value = formatPrice(1 / currentPrice, decimals);
-  }
-}
 </script>
 
 <template>
   <div>
     <div class="s-box p-4 space-y-3 pt-6">
```

Comment:
This create extra space for no reason
```suggestion
    <div class="s-box p-4 space-y-3">
```